### DISCUSSION TITLE
Why scramble groups endpoints based on  controllers instead of routes?

### DISCUSSION THREAD

**Discussion by moaoa** [dedoc/scramble#533] [Q&A] [answered]
Thanks for the awesome package :)

there is an issue with grouping endpoints, scramble will group endpoints based on controllers instead of routes.

for example i have a controller called AssignHomeworkController, in the docs there is a group named after this controller
![image](https://github.com/user-attachments/assets/583dd955-4b70-4913-b085-d06416a37260)

here is the group of routes: 

Route::prefix('teacher')->group(function () {
    Route::get('/quiz-students/{quizId}', UsersAttendingQuizController::class)->name('quiz-users');
    Route::post('/quiz-assignment', AssignQuizToGroupController::class)->name('assign-quiz');
    Route::post('/homework-assignment', AssignHomeworkToGroupController::class)->name('assign-homework');
    Route::post('/attendance/{lecture}', TakeAttendanceController::class)->name('attendance.take');
});

i want a group called teacher that have all of this routes, instead of grouping based on controllers.

**Comment by romalytvynenko** [comment, marked as answer]
@moaoa this is an awesome idea, never thought about it. I will explore about how to include it and not as a breaking change :-D

**Reply by chriswritests** [reply]
Any updates on this @romalytvynenko?

**Reply by romalytvynenko** [reply]
@chriswritests no update for now out of the box. There is `Scramble::resolveTagsUsing` method which can be used to override the default behavior.
